From 00db154df3450a83a429eb7a346f28ab33eeff70 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Sun, 10 Sep 2023 19:28:42 +0000
Subject: [PATCH 61/61] dts: add vqmmc-supply property to usdhc2 node

This property allows the actual state of the regulator (LDO5)
be displayed correctly over debug FS.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/som-imx8m-plus.dts  | 1 +
 arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dts  | 1 +
 arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/som-imx8m-plus.dts b/arch/arm64/boot/dts/compulab/som-imx8m-plus.dts
index f943ebff0204..34f3d5ccfc56 100644
--- a/arch/arm64/boot/dts/compulab/som-imx8m-plus.dts
+++ b/arch/arm64/boot/dts/compulab/som-imx8m-plus.dts
@@ -812,6 +812,7 @@ &usdhc2 {
 	pinctrl-1 = <&pinctrl_usdhc2_100mhz>, <&pinctrl_usdhc2_gpio>;
 	pinctrl-2 = <&pinctrl_usdhc2_200mhz>, <&pinctrl_usdhc2_gpio>;
 	cd-gpios = <&gpio2 12 GPIO_ACTIVE_LOW>;
+	vqmmc-supply = <&ldo5>;
 	vmmc-supply = <&reg_usdhc2_vmmc>;
         no-1-8-v;
 	bus-width = <4>;
diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dts
index e40e01074010..f6a8c41e045f 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dts
@@ -899,6 +899,7 @@ &usdhc2 {
 	pinctrl-1 = <&pinctrl_usdhc2_100mhz>, <&pinctrl_usdhc2_gpio>;
 	pinctrl-2 = <&pinctrl_usdhc2_200mhz>, <&pinctrl_usdhc2_gpio>;
 	cd-gpios = <&gpio2 12 GPIO_ACTIVE_LOW>;
+	vqmmc-supply = <&ldo5>;
 	vmmc-supply = <&reg_usdhc2_vmmc>;
 	bus-width = <4>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
index f2db6777a48f..8f2fb59991da 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
@@ -414,6 +414,7 @@ &usdhc2 {
 	pinctrl-1 = <&pinctrl_usdhc2_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc2_200mhz>;
 	bus-width = <4>;
+	vqmmc-supply = <&ldo5>;
 	status = "disabled";
 };
 
-- 
2.34.1

